src/brouwers/kitreviews/templates/kitreviews/review_form.html

```diff
@@ -1,57 +1,25 @@
-{% load i18n %}
+{% load i18n sniplates %}
 
-<div id="review_kit">
-    {% with reviewform.model_kit as field %}
-        {% include "kitreviews/review_form_field.html" %}
-    {% endwith %}
-</div>
+{% block form %}
 
-<div id="review_raw_text">
-    {% with reviewform.raw_text as field %}
-        {% include "kitreviews/review_form_field.html" %}
-    {% endwith %}
-</div>
+    <fieldset class="well">
+        {% load_widgets form='horizontal.form' %}
+        {% form_field form.model_kit %}
+        {% form_field form.raw_text %}
+        {% form_field form.album %}
+        {% form_field form.topic_id %}
+        {% form_field form.external_topic_url %}
 
-<div id="pos-neg-sides">
-    <div id="review_pos_points">
-        {% with reviewform.positive_points as field %}
-            {% include "kitreviews/review_form_field.html" %}
+        {% with label_grid_offset='col-sm-offset-2' %}
+            {% form_field form.show_real_name %}
         {% endwith %}
-    </div>
-    <div id="review_neg_points">
-        {% with reviewform.negative_points as field %}
-            {% include "kitreviews/review_form_field.html" %}
-        {% endwith %}
-    </div>
-</div>
-
-<div id="review_rating">
-    {% with reviewform.rating as field %}
-        {% include "kitreviews/review_form_field.html" %}
-    {% endwith %}
-</div>
 
-<div class="tab midpage">{% trans "Album, build report(s)" %}</div>
-
-<div id="review_links">
-    <p>
-        {% with reviewform.album as field %}
-            {% include "kitreviews/review_form_field.html" %}
-        {% endwith %}
-    </p>
-    <p>
-        {% with reviewform.topic_id as field %}
-            {% include "kitreviews/review_form_field.html" %}
+        {% with formset=properties %}
```

Comment:
I would put this in a different fieldset - you can end the current one here and start a new one. So then you have a fieldset with 'common' information (links to other data like topic, album, kit...), and a fieldset with the actual review.

For the actual review fieldset, start with the 'easy to enter data' -> so this formset with the properties. After that, move the `raw_text` field from the first fieldset to this one, after the properties.